public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -312,8 +354,16 @@ export const panelToRuleFormValues = async (
     return undefined;
   }
 
+  // TODO make this work with correct refids
   if (!queries.find((query) => query.datasourceUid === ExpressionDatasourceUID)) {
-    queries.push(getDefaultExpression(getNextRefIdChar(queries.map((query) => query.model))));
+    const [reduceExpression, _mathExpression] = getDefaultExpressions(getNextRefIdChar(queries), '-');
```

Comment:
💬 This is a bit weird, but the `getDefaultExpressions()` function wants the `refIds` and I don't have them yet, so I'm adding each expression one at a time to get the correct `refId`.